pkgs/by-name/ex/exo/package.nix

```diff
@@ -7,14 +7,15 @@
 }:
 python3Packages.buildPythonApplication rec {
   pname = "exo";
-  version = "0.0.9-alpha";
+  version = "0.0.10-alpha";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "exo-explore";
     repo = "exo";
-    tag = "v${version}";
-    hash = "sha256-H3Jxq0GS3Ha9cDsIHCsTupD0/Mp80Mxf2NvAyZytTb0=";
+    # TODO: git tag for 0.0.10-alpha starts with 'vv' instead of 'v'
```

Comment:
There is another tag without the extra v: https://github.com/exo-explore/exo/compare/vv0.0.10-alpha...v0.0.10-alpha